documentation/docs/install-pmm/install-pmm-server/baremetal/docker/run_with_vol.md

```diff
@@ -35,7 +35,7 @@ To run Docker with volume:
     docker exec -t pmm-server change-admin-password your_secure_password
     ```
 
```

Comment:
Here I'd mention that admin password can be also changed via UI when log in first time